packages/mdc-list/README.md

```diff
@@ -267,10 +268,10 @@ these should also receive `tabIndex="-1"`.
 #### Setup in `singleSelection()`
 
 When implementing a component that will use the single selection variant, the HTML should be modified to include
-the `aria-selected` attribute, the `mdc-list-item--selected` class should be added, and the `tabindex` of the selected
-element should be `0`. The first list item should have the `tabindex` updated to `-1`. The foundation method 
-`setSelectedIndex()` should be called with the initially selected element immediately after the foundation is 
-instantiated.
+the `aria-selected` attribute, the `mdc-list-item--selected` or `mdc-list-item--activated` class should be added, 
+and the `tabindex` of the selected element should be `0`. The first list item should have the `tabindex` updated 
```

Comment:
is it mandator for user to set `tabindex` -1 explicitly on first element? Doesn't list automatically sets tabindex to -1 to all other list items?

Reply:
If the user has marked an element as `selected`, they should render the first list element to `tabindex=-1`. The foundation may not be instantiated immediately when the page is loaded (such as in Wiz).